pkgs/applications/misc/golden-cheetah-bin/default.nix

```
@@ -0,0 +1,34 @@
+{ appimageTools, lib, fetchurl }:
```

```suggestion
{ appimageTools, lib, fetchurl, stdenv }:
```

Add stdenv for limiting architecture.